go.mod

```diff
@@ -98,3 +98,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20221109080512-912af81e4fa2
```

Comment:
Friendly reminder

Reply:
done in https://github.com/codeready-toolchain/registration-service/pull/286/commits/84cbc4505d1e12ed87d70335410f1b982a1a89eb